src/Trace/Integrations/Eloquent.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Trace\Integrations;
+
+/**
+ * This class handles instrumenting the Eloquent ORM queries using the stackdriver_trace extension.
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Trace\Integrations\Eloquent
+ *
+ * Eloquent::load();
+ */
+class Eloquent
+{
+    /**
+     * Static method to add instrumentation to the Symfony framework
+     */
+    public static function load()
+    {
+        if (!extension_loaded('stackdriver_trace')) {
+            return;
+        }
+
+        stackdriver_trace_method('Illuminate\Database\Eloquent\Builder', 'getModels', function ($scope, $columns) {
```

Comment:
We could, but it will crash if the user doesn't have Eloquent as a dependency and we aren't adding the dependency to this library. We could wrap the whole thing in a check to see if the class is available, but as a string, it's ok if the class doesn't exist.

Additionally, for integrations like Doctrine where the class namespace changed between versions, we'd then want to split into 2 classes which would be ok.